instruqt-tracks/sentinel-for-terraform-v4/05-restrict-domains-v2/assignment.md

```diff
@@ -0,0 +1,109 @@
+---
+slug: restrict-domains-v2
+id: sxafq5mq9ylk
```

Comment:
Let's clear the `id:` fields out of `assignment.md` while we're moving to v3 of the track config